hdf5.spec

```
@@ -589,6 +583,9 @@ done
 %endif
 
 %changelog
+* Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1.g07066a381e
+- Update HDF5 to version 1.12.0
+
```

Major functionality changes like dropping the examples should be detailed here when they happen.  If we can resolve that easily enough, then this `%changelog` entry is fine, but if we do end up dropping them, that should be added here.
```suggestion
- Dropped packaging of examples

```